ingestors/cli.py

```diff
@@ -112,5 +113,11 @@ def debug(path, languages=None):
         pprint(entity.to_dict())
 
 
+@cli.command()
+def cache_clear():
+    """Delete all ingest cache entries."""
+    Tags("ingest_cache").delete()
```

Comment:
I think ideally you’d want to be able to clear only a subset of the cache based on key prefixes, e.g. only delete entries starting with `ocr:` (but not entries starting with `pdf:`.

Reply:
@tillprochaska I added a prefix argument.